packages/orchestration/src/cosmos-api.ts

```diff
@@ -97,6 +100,12 @@ export type CosmosChainInfo = Readonly<{
   stakingTokens?: Readonly<Array<{ denom: string }>>;
 }>;
 
+/** @see {QueryDelegationTotalRewardsResponse} */
+export interface OrchestrationQueryDelegationTotalRewardsResponse {
+  rewards: { validatorAddress: string; reward: DenomAmount[] }[];
```

Comment:
nb: Consider
```suggestion
  rewards: { validator: CosmosValidatorAddress; amount: DenomAmount[] }[];
```
for consistency, although we don't need this response for `withdrawReward` or `withdrawRewards`